webpack/components/SelectOrg/SelectOrg.scss

```diff
@@ -0,0 +1,5 @@
+#reactRoot {
+  .well {
```

Comment:
We probably want to make this more specific to the select organization page.  I doubt we want a 30px margin on all wells.